ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/ConversionsSpec.scala

```diff
@@ -20,72 +20,66 @@ import com.daml.lf.transaction.{BlindingInfo, NodeId}
 import com.daml.lf.value.Value.ContractId
 import org.scalatest.{Matchers, WordSpec}
 
-import scala.collection.immutable.ListSet
-
 import scala.collection.JavaConverters._
+import scala.collection.immutable.{ListMap, ListSet}
 
 class ConversionsSpec extends WordSpec with Matchers {
   "Conversions" should {
     "correctly and deterministically encode Blindinginfo" in {
-      encodeBlindingInfo(blindingInfoNotInOrder) shouldBe orderedEncodedBlindingInfo
+      encodeBlindingInfo(wronglySortedBlindingInfo) shouldBe correctlySortedEncodedBlindingInfo
```

Comment:
Based on what the `encodeBlindingInfo` function does the actual properties that matter here are 1) we encode the right set of parties and 2) the elements are sorted. I believe this test case would be much easier to comprehend if we checked those properties explicitly (e.g., using `theSameElementsAs` then `sorted` assertions).